src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -110,11 +112,13 @@ const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> = p
         />
         <QueryFilter {...rest} />
         <QueryFilterToggleButton />
-        <FacetNav
-          facetsToPlot={facetsToPlot}
-          showNotch={false}
+        <TotalQueryResults
           token={props.token}
+          frontText={''}
+          showNotch={false}
+          {...rest}
         />
```

Comment:
Was in the FacetNav component before, but now pulled out (the two components were separated visually in PORTALS-1820)